app/src/main/java/ar/com/wolox/android/example/test/mocks/AuthenticationServiceMock.kt

```diff
@@ -0,0 +1,45 @@
+package ar.com.wolox.android.example.test.mocks
+
+import ar.com.wolox.android.example.externalServices.AuthenticationService
+import ar.com.wolox.android.example.model.User
+import ar.com.wolox.android.example.utils.Extras.AuthenticationCredentials.EMAIL
+import ar.com.wolox.android.example.utils.Extras.AuthenticationCredentials.PASSWORD
+import ar.com.wolox.android.example.utils.Extras.AuthenticationCredentials.WRONG_EMAIL
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.mock
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class AuthenticationServiceMock {
+
+    companion object {
```

Comment:
just a convention. Companion object goes at the very end of the class. Is the "last" declaration

Reply:
Is it ok here? It's at the bottom of the class, but it has a lot of methods inside it 😅